src/platformDependent.cpp

```diff
@@ -35,3 +36,28 @@ std::string shasta::timeoutCommand()
 #endif
     
 }
+
+uint64_t shasta::getPeakMemoryUsage() {
+    uint64_t peakMemoryUsage = 0ULL;
+#ifdef __linux__
+    ifstream procStats("/proc/self/status");
+    if (procStats) {
+        string line;
+        while (std::getline(procStats, line)) {
+            if (string::npos == line.find("VmPeak")) {
+                continue;
+            }
+            size_t pos = line.find(":");
+            while (pos < line.size() && !isdigit(line[pos])) {
+                pos++;
+            }
+            char* end;
+            peakMemoryUsage = std::strtoull(line.c_str() + pos, &end, 10);
+            // Convert from kB to bytes.
+            peakMemoryUsage *= 1024;
+            break;
+        }
+    }
+#endif
+    return(peakMemoryUsage);
```

Comment:
The parentheses are unnecessary and are generally not used in `return` statements in Shasta.